### PR TITLE
Update translations (Korean)

### PR DESCRIPTION
**Pull request by Mrkim-0**
Describe the changes you have conducted here

Preview the messages on https://inlang.com/github.com/Mrkim-0/pokemonAutoChess .

Added Korean Language made 74 changes 

Changed eng root -> ingrain

### CODE REVIEW

File: project.inlang/settings.json [hunk 1]
```diff
@@ -1,10 +1,11 @@
 {
   "$schema": "https://inlang.com/schema/project-settings",
-  "sourceLanguageTag": "en",
+  "sourceLanguageTag": "ko",
```
Comment (sylvainpolletvillard) [inline project.inlang/settings.json:3]: please keep english as sourceLanguageTag

**Comment by sylvainpolletvillard** [comment]
I don't know why so many labels were reordered in other languages. It's a bit annoying to review, we may only pick your korean translation file

**Comment by Mrkim-0** [comment]
I must have messed up something with the setting. I wasn't so familiar with
the tool. Please just pick my korean translation file.

**Comment by sylvainpolletvillard** [comment]
Closing in favor of https://github.com/keldaanCommunity/pokemonAutoChess/pull/1910

**Pull request closed without merging**